public/index.html

```diff
@@ -16,6 +19,7 @@
     <div id="root">
 
     </div>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/toastr.min.js"></script>
```

Comment:
Great work on using the CDN link as it loads the resources faster

Reply:
Thank you